native/src/window/action.rs

```diff
@@ -37,6 +37,8 @@ pub enum Action<T> {
     ToggleMaximize,
     /// Fetch the current [`Mode`] of the window.
     FetchMode(Box<dyn FnOnce(Mode) -> T + 'static>),
+    /// Fetch the window id from winit
```

Comment:
We can't mention `winit` in `iced_native`, since it's a downstream dependency!

I would just say: `Fetch the id of the window`, and maybe explain the meaning of the `u64` a bit.